cmd/check.go

```diff
@@ -10,8 +10,7 @@ import (
 	"github.com/opdev/opcap/internal/capability"
 	"github.com/opdev/opcap/internal/operator"
 
-	pkgserverv1 "github.com/operator-framework/operator-lifecycle-manager/pkg/package-server/apis/operators/v1"
-
+	pkgserversv1 "github.com/operator-framework/operator-lifecycle-manager/pkg/package-server/apis/operators/v1"
```

Comment:
Why this change ? More importantly, the identifier of this package is inconsistent through the codebase:

- For instance (but not only) `cmd/package.go`, it's named `pkgserverv1`
- Here and in `internal/packages/packages.go`, it's named `pkgserversv1`

Reply:
Inadvertent change. I'll fix it.